### DISCUSSION TITLE
Update notifications and telemetry feedback

### DISCUSSION THREAD

**Discussion by timschumi** [timschumi/gmod-chttp#29] [General]
Starting with version 1.10.0, CHTTP collects some light telemetry data as described [here](https://github.com/timschumi/gmod-chttp#update-notifications-and-telemetry).

To reiterate, this is just to satisfy my curiosity about how fast updates propagate and what systems the module is ran on.

Feel free to leave any feedback regarding either of these features here!

**Comment by FlorianLeChat** [comment]
I think it would also be a good idea if you could share these data publicly for educational purposes, I'd personally be curious to know if many servers are running the 64-bit branch or if any Garry's Mod servers are still running on Windows.

**Reply by timschumi** [reply]
I'll look into it.

While I'm generally not opposed to it, I still have to figure out whether I can just make the Grafana dashboard public without risking granting access to other data that isn't supposed to be public. If that doesn't work, I'll probably just put up an endpoint that generates the statistics in text format.

I'm also concerned that having a public counter will lead to people spamming fake metrics, which I have no real way of validating.

**Reply by FlorianLeChat** [reply]
I totally understand, if you think this involves too much work/risk in comparison to your initial purpose, that's no big deal.

**Reply by timschumi** [reply]
Select aggregated statistics over the last week are now available [here](https://chttp.timschumi.net/statistics.json).

To everyone: Feel free to (moderately) scrape, as long as you make sure to send me the links to any cool visualizations you might make. :^)
